eng/DependencyValidation/DependencyValidation.modelproj

```diff
@@ -34,6 +34,26 @@
     </Validate>
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\modules\try-convert\src\MSBuild.Abstractions\MSBuild.Abstractions.csproj">
```

Comment:
This may become clearer later, but why are all these additional references needed now?

Reply:
Not really sure. It auto-added them and I'm not really sure how this project file works

Reply:
The modelproj format still uses the old style format, so......

Reply:
Got it.